sass/base/_structure.scss

```diff
@@ -142,6 +142,7 @@ body.app__body #root {
     }
 
     @media screen and (max-width: 768px) {
+        overflow-y: visible;
```

Comment:
Making this change at this high level, will this affect other screens where we don't want this behavior (don't have an example in mind, just wondering)?

Reply:
good point, moving it to the preparing_workspace file, so we have a reduced scope